From 5c260d64c5a09f84c23d39f80f06bdcbe393e1b8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Piotr=20Kr=C3=B3l?= <piotr.krol@3mdeb.com>
Date: Wed, 4 Apr 2018 14:41:58 +0200
Subject: [PATCH 1/9] PcAtChipsetPkg/PcRtc: set all bits in register D except
 VRT
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Piotr Król <piotr.krol@3mdeb.com>
---
 PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c b/PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c
index c032e16217c3..f3f2bb08697e 100644
--- a/PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c
+++ b/PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c
@@ -144,10 +144,10 @@ PcRtcInit (
   RtcRead (RTC_ADDRESS_REGISTER_C);
 
   //
-  // Clear RTC register D
+  // Clear RTC register D - all bits except VRT (Valid RAM and Time)
   //
   RegisterD.Data = FixedPcdGet8 (PcdInitialValueRtcRegisterD);
-  RtcWrite (RTC_ADDRESS_REGISTER_D, RegisterD.Data);
+  RtcWrite (RTC_ADDRESS_REGISTER_D, RegisterD.Data & 0x80);
 
   //
   // Wait for up to 0.1 seconds for the RTC to be updated
-- 
2.17.1

